apps/condo/domains/notification/access/RegisterPushNotificationTokenService.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generated by `createservice notification.RegisterPushNotificationTokenService --type mutations`
+ */
+const get = require('lodash/get')
+
+async function canRegisterPushNotificationToken (params) {
+    if (get(params, 'authentication.item.isAdmin')) return true
```

Comment:
It might be better to use an existing style

Reply:
this way is much safer for deep nested attrs, and way more readable. or we should upgrade node version and use `?.` instead